README.md

```diff
@@ -154,6 +154,12 @@ We have recently changed the destination service to which these metrics are repo
 
 You can easily opt-out of metrics collection by adding `opt_out_usage` at the top of your `Fastfile` or by setting the environment variable `FASTLANE_OPT_OUT_USAGE`. Participating helps us provide the best possible support for _fastlane_, so we hope you'll consider it a plus! :heavy_plus_sign:
 
+## Crash Reporting
+
+In order to continuously improve stability, _fastlane_ will record crash reports with sanitized stacktraces. Sanitization removes all personal information from the stacktrace and error message (including home directories, _fastlane_ path, gem paths, environment variables, and parameters).
```

Comment:
Make the second sentence less absolute.  Maybe just remove "all" from "removes all personal information".